thirdeye/thirdeye-frontend/app/pods/components/self-serve-alert-yaml-details/template.hbs

```diff
@@ -1,22 +1,24 @@
 <div class="te-search-results__header">
   <div class="te-search-results__title-group">
     <div class="te-search-results__title">
-      <span title={{alertData.detectionName}}>{{alertData.detectionName}}</span>
+      <h3 class="te-search-results__title-name" title={{alertData.detectionName}}>{{alertData.detectionName}}</h3>
```

Comment:
it's weird to see the title attribute in `h` tag. If we want to support the screenreader, we may set a rule to add all title attribute for h tag.

Reply:
In this case, it's used for displaying the full title on hover when it's truncated.

Reply:
That's great. I curious before why you use the same `alertData.detectionName` twice in the tag of text and title field. If the class name can hint truncated, It would better to use `te-search-results__title-name--truncated`. When users read HTML code, they will easy to know the usage.

Reply:
the class name doesn't hint the truncated text (there are no handlers), instead, the browser is using the title tag to display native tooltip on hover regardless if the text is truncated or not)
![truncated-title](https://user-images.githubusercontent.com/991587/97604454-e4184b00-19e3-11eb-858a-e9611d5e1572.png)